lib/parseTriggersEmulator.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var FirebaseError = require('./error');
+var fork = require('child_process').fork;
+var path = require('path');
+var RSVP = require('rsvp');
+
+var TRIGGER_PARSER = path.resolve(__dirname, './triggerParser.js');
+
+module.exports = function(firebaseConfig, projectId, dbInstance, sourceDir) {
+  return new RSVP.Promise(function(resolve, reject) {
+    var env = {
+      GCLOUD_PROJECT: projectId,
+      DB_NAMESPACE: dbInstance,
```

Comment:
FIREBASE_PROJECT doesn't exist in the runtime yet, so for now it is just used for the purpose of emulating functions.config().firebase